### PR TITLE
Add Camo Studio

### PR DESCRIPTION
**Pull request by peterlewis**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

**Comment by miccal** [comment]
You need to escape the square brackets `[` and `]`:
```
-bash-5.0.18- /Users/miccal (29) [> curl -I 'https://reincubate.com/res/labs/camo/Camo%20Studio%201.0.1%20(53)%20[Release].app.zip'
curl: (3) bad range in URL position 70:
https://reincubate.com/res/labs/camo/Camo%20Studio%201.0.1%20(53)%20[Release].app.zip
                                                                     ^
```
```
-bash-5.0.18- /Users/miccal (29) [> curl -I 'https://reincubate.com/res/labs/camo/Camo%20Studio%201.0.1%20(53)%20\[Release\].app.zip'
HTTP/1.1 200 OK
Date: Fri, 17 Jul 2020 00:58:58 GMT
Content-Type: application/zip
Content-Length: 10342151
Connection: keep-alive
Set-Cookie: __cfduid=dd8d81edba7f2789653a2f0bad9ab2a781594947538; expires=Sun, 16-Aug-20 00:58:58 GMT; path=/; domain=.reincubate.com; HttpOnly; SameSite=Lax
Last-Modified: Thu, 16 Jul 2020 11:50:15 GMT
ETag: "5f103ef7-9dcf07"
Accept-Ranges: bytes
CF-Cache-Status: DYNAMIC
cf-request-id: 03fbe124900000050728822200000001
Expect-CT: max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
Strict-Transport-Security: max-age=2592000
Server: cloudflare
CF-RAY: 5b400480e84b0507-LAX
alt-svc: h3-27=":443"; ma=86400, h3-28=":443"; ma=86400, h3-29=":443"; ma=86400
```

File: Casks/camo-studio.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'camo-studio' do
+  version '1.0.1,53'
+  sha256 '3da0ac5fbfea28848de0ee7fd5832298884e241a189d56150013dfebfba3589d'
+
+  url "https://reincubate.com/res/labs/camo/Camo%20Studio%20#{version.before_comma}%20(#{version.after_comma})%20[Release].app.zip"
```
Comment (miccal) [inline Casks/camo-studio.rb:5]: Please add the following `appcast`:
```
appcast 'https://uds.reincubate.com/release-notes/camo/?format=sparkle'
```

Comment (peterlewis) [reply Casks/camo-studio.rb]: Roger that! Thanks!

**Comment by peterlewis** [comment]
> You need to escape the square brackets `[` and `]`:

Thanks for that. I've amended but still failing with the same error. Would appreciate suggestions. Many thanks!

**Comment by miccal** [comment]
Oh yeah, forgot that the macOS curl is different -- try this:
```
url "https://reincubate.com/res/labs/camo/Camo%20Studio%20#{version.before_comma}%20%28#{version.after_comma}%29%20%5BRelease%5D.app.zip"
```

**Comment by peterlewis** [comment]
> Oh yeah, forgot that the macOS curl is different -- try this:

Ah splendid, many thanks, that's more like it! Right, I think this is ready now. Sod's law they'll have updated the download link to a more structured URL by the time I get this ruddy PR in! :)

**Pull request merged**